React/Base/RCTUtils.m

```diff
@@ -286,6 +286,31 @@ id RCTNilIfNull(id value)
   return value == (id)kCFNull ? nil : value;
 }
 
+NSURL *RCTWebSocketExecutorURL(NSString *appendString)
+{
+  NSMutableString* url;
+  NSDictionary *dict = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"RCTWebSocketExecutor"];
+  if( dict )
+  {
+    NSString* protocol = [dict objectForKey:@"protocol"];
+    NSString* hostname = [dict objectForKey:@"hostname"];
+    NSString* port = [NSString stringWithFormat:@"%@", [dict objectForKey:@"port"]]; ;
+    url = [NSMutableString stringWithString:protocol];
+    [url appendString:@"//"];
+    [url appendString:hostname];
+    [url appendString:@":"];
+    [url appendString:port];
```

Comment:
String concats are very error prone. If you take my suggestion about URL in `Info.plist` you can just do `[NSURL URLWithString]` and then `[url URLByAppendingPathComponent:]`